### PR TITLE
Installations for react apps

### PR DESCRIPTION
**Pull request by geekyrajshri**
**Description**

### CODE REVIEW

File: src/pages/react/Installations/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: React App Dependencies Installation 
+---
+## Pre-requisites 
+We need the right combinations of `Babel` ,` Webpack` and `NPM `dependencies to get started with a basic React Application.  `create-react-app` a command line tool is provided by React developers to abstract all the complexities involved in setting up a react project.
+What goes under the hood when we use this command is what we will be explaining here.
+
+```C
```
Comment (Bouncey) [inline src/pages/react/Installations/index.md]: This does not need to be in a code block. Simply using `1.` `2.` `3.` will render out the correct formatting for a list, like so:

1. one thing
2. another thing
3. a really awesome thing

You can see how formatting is added automatically through markdown

File: src/pages/react/Installations/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+1.	Create an empty directory and change the directory .
+2.	Run `npm init` command that will ask  a bunch of questions, and then write a package.json for us.
+3.	Now run the npm command to install `react` and `react-dom` which will be stored in `node_modules` folder.
+      
+```
+Here are the terminal commmands:-
+
+```C
```
Comment (Bouncey) [inline src/pages/react/Installations/index.md]: This should be in a code block with `shell` formatting.

Please change this to ` ```shell` or ` ```sh`

File: src/pages/react/Installations/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+$ mkdir react_app
```
Comment (Bouncey) [inline src/pages/react/Installations/index.md]: Please remove  all `$`'s

This text is already in a code block and doesn't require the `$` to denote it is a shell command. It only confuses newer developers who try to copy/paste the whole line

File: src/pages/react/Installations/index.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+title: React App Dependencies Installation 
+---
+## Pre-requisites 
+We need the right combinations of `Babel` ,` Webpack` and `NPM `dependencies to get started with a basic React Application.  `create-react-app` a command line tool is provided by React developers to abstract all the complexities involved in setting up a react project.
+What goes under the hood when we use this command is what we will be explaining here.
```
Comment (Bouncey) [inline src/pages/react/Installations/index.md]: This is a fairly bold statement to have, and by the end of the article I would not have a clue what was going on. All I would have is a bunch of dependencies in a `node_modules` folder.

You have not described how to hook up babel + webpack + react + react-dom + jest + eslint + all the other things that `create-react-app` does for us.

**Comment by imcodingideas** [comment]
@geekyrajshri it would be awesome if you could rename the folder from `Installations` to `installations` to get rid of the capital case **I**.

**Comment by HKuz** [comment]
I closing this PR as stale - it looks like another PR explaining create react app was accepted in the `/installation` folder. Feel free to add to that article to better explain what's going on under the hood! Happy coding! 🎉 🎃 👻

**Pull request closed without merging**